common/client_socket.h

```diff
@@ -14,13 +14,13 @@
  * limitations under the License.
  */
 
-#ifndef CDC_RSYNC_CLIENT_SOCKET_H_
-#define CDC_RSYNC_CLIENT_SOCKET_H_
+#ifndef COMMON_CLIENT_SOCKET_H_
```

Comment:
I think usually you would like to have a common top-level prefix for your whole project. But I guess as "common" is only intended to be used within your project, it is fine.

Reply:
Thanks, yeah, we don't intend this to be used as a library.